src/main/docs/guide/whatsNew.adoc

```diff
@@ -1,3 +1,6 @@
+=== Micronaut Data 4.0
```

Comment:
We can probably release it in the next minor version, it might take some time for Micronaut 4.

Reply:
Removed it https://github.com/micronaut-projects/micronaut-data/pull/1827/commits/dad9e447ba600695b18fe4b6bcc68e9a3cefae6a
We have `@since 4.0.0` throughout the code but that can be updated later I guess.